### PR TITLE
Add a script to create key for new users

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

- Adds Outline VPN API functions and a function to generate vpn key for new hires.

Fixes #679 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kelvinkipruto** [comment]
@koechkevin When testing this, do you use a hosted Outline Manager or can a locally run work?

**Comment by koechkevin** [comment]
> @koechkevin Is there a test Outline Sever you are using for this?

It uses the existing server, but we have email addresses prefixed with test- so that a clean up will happen

**Comment by koechkevin** [comment]
@kelvinkipruto @thepsalmist @VinneyJ

File: apps/vpnmanager/src/lib/email/sender.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+interface MailSender {
+  recipient: string;
+  key: string;
+}
+
+export async function sendVpnKeyEmail({ recipient: to, key }: MailSender) {
+  try {
+    const sendGridApiKey = process.env.VPN_MANAGER_SENDGRID_API_KEY as string;
+    if (!sendGridApiKey) {
```
Comment (thepsalmist) [inline apps/vpnmanager/src/lib/email/sender.ts:14]: Mostly looks good to me @koechkevin, on error handling I see in certain cases we're throwing the actual error & others returning `null`.  Isn't it better to explicitly throw the errors + message

**Review by thepsalmist — APPROVED** [review]

**Review by VinneyJ — APPROVED** [review]
Looks good!

**Review by kelvinkipruto — APPROVED** [review]

**Pull request merged**